Detectors/MUON/MCH/Calibration/src/pedestal-decoding-workflow.cxx

```diff
@@ -140,6 +163,7 @@ class PedestalsTask
   void reset()
   {
     mDigits.clear();
+    mErrors.clear();
```

Comment:
I guess this is actual fix advertised in the PR title, right ? That one and the addition of the ERRORS output make perfect sense.

But what about the patchPage change : is it needed / related to this fix at all ?

Reply:
The patchPage change is not related to the memory leak, but introduces the same code we have in the standard decoder. I just wanted to avoir two independent PR with small changes...

Reply:
As long as it is *actually* needed and not some leftover, that's fine with me then.